docs/topics/install-olympia/installation.rst

```diff
@@ -177,18 +176,47 @@ It's awesome, you want it.
 The file ``local_settings.py`` is for local use only; it will be ignored by
 git.
 
+.. _installation-database:
 
 Database
 --------
 
-The database is initialized automatically using the `make full_init` command
-you saw earlier. If you want to start from scratch and recreate the database,
-you can just run the `make initialize_db` command.
+By default, Olympia connects to the ``olympia`` database running on
+``localhost`` as the user ``root``, with no password. To create a database,
+run::
 
-This will also fake all the `schematic`_ migrations, and allow you to create a
-superuser.
+    $ mysql -u root -p
+    mysql> CREATE DATABASE olympia CHARACTER SET utf8 COLLATE utf8_unicode_ci;
 
-.. _schematic: https://github.com/mozilla/schematic
+
+If you want to change settings, you can either add the database settings in
+your :ref:`local_settings.py<example-settings>` or set the environment variable
+``DATABASE_URL``::
+
+    export DATABASE_URL=mysql://<user>:<password>@<hostname>/<database>
+
+If you've changed the user and password information, you need to grant permissions
+to the new user::
+
+    $ mysql -u root -p
+    mysql> GRANT ALL ON olympia.* TO olympia@localhost IDENTIFIED BY '<YOUR_PASSWORD>';
```

Comment:
I think you need to replace `olympia@localhost` by `<YOUR_USER>@localhost`, or explain that the example is for the user named `olympia`